backend/src/Kernel.php

```diff
@@ -41,6 +45,15 @@ public function getProjectDir(): string
         return \dirname(__DIR__);
     }
 
+    public function process(ContainerBuilder $container): void
+    {
+        $isJaasActive = $container->resolveEnvPlaceholders('%env(JAAS_ACTIVE)%', true);
```

Comment:
I think it is more readable if you add a new parameters in the services.yaml file pointing to this env var, in this file you can use $container->getParameter()

Reply:
I wasn't sure of adding the parameters.yml, because we dont have one. I will do it. Do I need to add the extra script in composer.json to generate it?

Reply:
You don't need a parameters.yml, you can add a parameter on any .yml (better if you use services.yaml)

Reply:
Genius! gotcha